packages/lib-core/src/utils/k8s/k8s.ts

```diff
@@ -0,0 +1 @@
+export const k8sBasePath = `${window.SERVER_FLAGS.basePath}api/kubernetes`;
```

Comment:
Oooh, we don't want this. But I bet this is needed for the apiDiscovery 😕 

We need to rely maybe on the appFetch function in this case so we can fetch to the same API backend that the k8s utils will call 🤔 

cc @vojtechszocs 

---

This can be deleted for now... but we'll need to think on this with the work @vojtechszocs has in https://github.com/openshift/dynamic-plugin-sdk/pull/30